tests/client/core/components/TestSearchResults.js

```diff
@@ -48,11 +59,25 @@ describe('<SearchResults />', () => {
         { name: 'result 2', slug: '2' },
       ],
     });
-    const searchResultsMsg = root.message;
-    assert.include(searchResultsMsg.textContent, 'Your search for "test" returned 5 results');
+    const searchResultsMessage = root.message;
+    assert.include(searchResultsMessage.textContent,
+                   'Your search for "test" returned 5 results');
 
     const searchResultsList = root.results;
     assert.include(searchResultsList.textContent, 'result 1');
     assert.include(searchResultsList.textContent, 'result 2');
   });
+
+  it('renders search results in the singular', () => {
+    const root = renderResults({
+      count: 1,
+      query: 'test',
+      results: [
+        { name: 'result 1', slug: '1' },
+      ],
+    });
+    const searchResultsMessage = root.message;
+    assert.include(searchResultsMessage.textContent,
```

Comment:
The aligned indent is occasionally a bit nicer, but I think the win of the two-space indent is that it's:

1. never wasteful
2. never *really* ugly